<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Policy Dialog</string>
  </property>
  <widget class="QLineEdit" name="policyEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>policy:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="attributeList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="andButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>and</string>
   </property>
  </widget>
  <widget class="QPushButton" name="orButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>or</string>
   </property>
  </widget>
  <widget class="QPushButton" name="notButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>not</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sureButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>sure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftBracketButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightBracketButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>6</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Availabel attributes:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
